README.md

```diff
@@ -29,7 +29,7 @@ Your answer...
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The request hits the router of an application. If the router recognizes the route then it sends the request to the controller. From there the controller handles things. If the router doesn't recognize the route, then you get an error and the request does not make it to the controller.
 ```
```

Comment:
:+1: although to be explicit, the router maps a **request** to a particular controller action. A **route** is what defines that mapping.